styles/components/_modalWindow.mediaQueries.scss

```diff
@@ -0,0 +1,261 @@
+//TO REMOVE!!!!
+html {
+  overflow: hidden;
+}
+
+@media only screen and (max-width: $breakpoint-xxl-max) {
+  .modal {
+    background-color: yellowgreen;
+    &__container {
+      height: calc(100% - 124px);
+      max-width: unset;
+      width: calc(100% - 124px);
+      padding: 32px;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+    }
+    &__heading {
+      font-size: 32px;
+    }
+    &__text {
+      font-size: 24px;
+      margin: 0;
+    }
+  }
+  .details {
+    margin-left: 20%;
+    padding-top: 0;
+    min-height: 200px;
+    &__heading {
+      font-size: 24px;
+      margin: 16px 0 8px 0;
+      text-align: left;
+      padding-left: 16px;
+    }
+    &__table {
+      border-collapse: collapse;
+      display: block;
+      height: calc(100% - 62px);
+      overflow-y: scroll;
+      & tbody {
+        display: table-row-group;
+      }
+      & thead tr {
+        display: table-row;
+      }
+      & thead {
+        padding-left: unset;
+      }
+      & tbody {
+        height: unset;
+        border-spacing: unset;
+      }
+      & th {
+        display: table-cell;
+      }
+      & th:first-of-type,
+      & td:first-of-type {
+        padding: 4px 4px 4px 16px;
+        text-align: left;
+      }
+      & td,
+      & th {
+        width: 30%;
+        padding: 8px;
+      }
+      &__image {
+        & img {
+          border-radius: 8px;
+          filter: drop-shadow(0px 2px 2px rgba(0, 0, 0, 0.25))
+            drop-shadow(2px 2px 8px rgba(255, 0, 0, 0.9));
+        }
+      }
+    }
+  }
+  .player-form {
+    padding: 0;
+    &__item,
+    &__label {
+      flex: 1;
+    }
+    &__input {
+      height: 64px;
+      padding-left: 12px;
+      font-size: 24px;
+      width: 100%;
+    }
+    &__label {
+      font-size: 24px;
+      padding: 0 0 0 16px;
+    }
+    &__button {
+      margin-top: 16px;
+      width: 100%;
+      font-size: 24px;
+      line-height: 32px;
+      border-radius: 8px;
+    }
+    &__image {
+      opacity: 0.8;
```

Comment:
don't set opacity on elements, compare with figma hi-fi design and apply fixes

Reply:
It is moved under the text on smaller screens, it would be hard to read if we left it as it is.